test/e2e/framework/bindata_util.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	"os"
+	"path/filepath"
+
+	"k8s.io/kubernetes/test/e2e/generated"
+)
+
+const (
+	// Make our own subdirectory for just-in-time generated data files.
+	GENERATED_ASSETS = "TEST_E2E_GENERATED_ASSETS_FOLDER"
+)
+
+// This file depends on an Asset() function being available which is generated by go-bindata.
+//
+// To generate these assets, you essentially need to do the following steps (should automate eventually).
+//
+// 	go get -u github.com/jteeuwen/go-bindata/...
```

Comment:
Why not just create a script for this in hack.  generate-e2e-examples.sh or some such, and copy the existing pattern there.